modules/ROOT/pages/performance-and-impact.adoc

```diff
@@ -0,0 +1,19 @@
+= Anypoint Monitoring Performance and Impact
+
+These are the findings from Anypoint Monitoring performance testing. 
+
+== Mule Monitoring Agent Performance Impact
+
+Expected CPU utilization impact: 3-5%
+
+== Data Retention and Aggregation
+
+* Minute Resolution for Metrics
+* Free tier (non-Titanium)
+ ** Metrics retention is 30 days for multi-tenant
+
+== Screen Resolution
```

Comment:
Feels like this should be a note or subhead of the intro, not after the rich content about what the product does. I know this is early draft, so not sure this is relevant.